src/constants/index.js

```diff
@@ -55,4 +55,6 @@ export const OTHER_MAIN_TYPES = [
   'C2991', //Disease or Disorder (Other Disease)
 ];
 
-export const NIH_ZIPCODE = "20892";
```

Comment:
why in constants.js and not events?

Reply:
It serves two purposes:
1. The constant that determines which "start over" type link was clicked on (`Start Over` or `Try a new search`).
2. To serve as the event source

Thought it was most appropriate to put it here mainly for `1` above, but could move it to somewhere else 'cos of `2` like you noted.